### ISSUE TITLE
[Guidance required] What do we do with screenshots?

### ISSUE THREAD

**Issue by techbelly** [dxw/brandbook#26]
I've noticed we occasionally show website screenshots with mobile phone chrome, pshopped on to laptop screens, or just shown bare in the slides. 

Would be good to have guidance on what we do here.

**Comment by gazaston** [comment]
Yes, good shout. I believe Leeky has some in-situ backgrounds we could use? @leeky — am I right? If so, can you point us in the direction of them and I'll add something to the guidelines?
